e2e-tests/ukef/cypress/e2e/journeys/portal/amendments/journeys/change-both-cover-end-date-and-facility-value.spec.js

```diff
@@ -0,0 +1,139 @@
+import relative from '../../../../relativeURL';
+import MOCK_USERS from '../../../../../../../e2e-fixtures/portal-users.fixture';
+import { MOCK_APPLICATION_AIN_DRAFT } from '../../../../../../../e2e-fixtures/gef/mocks/mock-deals';
+import { anIssuedCashFacility } from '../../../../../../../e2e-fixtures/mock-gef-facilities';
+import { applicationPreview } from '../../../../../../../gef/cypress/e2e/pages';
+import whatDoYouNeedToChange from '../../../../../../../gef/cypress/e2e/pages/amendments/what-do-you-need-to-change';
+import coverEndDate from '../../../../../../../gef/cypress/e2e/pages/amendments/cover-end-date';
+import doYouHaveAFacilityEndDate from '../../../../../../../gef/cypress/e2e/pages/amendments/do-you-have-a-facility-end-date';
+import facilityEndDate from '../../../../../../../gef/cypress/e2e/pages/amendments/facility-end-date';
+import eligibility from '../../../../../../../gef/cypress/e2e/pages/amendments/eligibility';
+import facilityValue from '../../../../../../../gef/cypress/e2e/pages/amendments/facility-value';
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+context('Amendments - Change both cover end date and facility value - full journey', () => {
+  /**
+   * @type {string}
+   */
+  let dealId;
+
+  /**
+   * @type {string}
+   */
+  let facilityId;
+  /**
+   * @type {string}
+   */
+  let amendmentId;
+  /**
+   * @type {Date}
+   */
+  const mockFacility = anIssuedCashFacility({ facilityEndDateEnabled: true });
```

Comment:
```suggestion
  const mockFacility = issuedCashFacility({ facilityEndDateEnabled: true });
```

Reply:
This is used in many many places (not named in this file) so not changing here (as it will cause many files changed in this PR).

Reply:
Thanks I'll raise a separate PR to rename this.